docs/GUIDES.md

```diff
@@ -73,7 +73,118 @@ The playing field is a grid of tiles, addressable like a coordinate system.
    │ 0│ 1│ 2│ 3│ 4│ 5│ 6│ 7│ 8│ 9│10│11│12│13│14│15│16│17│18│19│20│
 </pre>
 
-## Writing Code
+## Tutorial
+
+### 0. Prerequisites
+
+- explain what Scenic is
```

Comment:
I was thinking, maybe the elixir section above should be here...

Reply:
If I remember well from the last meeting, the elixir section should be moved in a separated guide.

https://github.com/codecurious-bln/elixir-orga-team/blob/master/meetings/2019-10-29.md#outcome